mlflow/tensorflow/__init__.py

```diff
@@ -919,24 +920,30 @@ def _get_tensorboard_callback(lst):
 _TensorBoardLogDir = namedtuple("_TensorBoardLogDir", ["location", "is_temp"])
 
 
-def _setup_callbacks(lst, metrics_logger):
+def _setup_callbacks(callbacks, metrics_logger):
     """
     Adds TensorBoard and MlfLowTfKeras callbacks to the
     input list, and returns the new list and appropriate log directory.
     """
     # pylint: disable=no-name-in-module
     from mlflow.tensorflow._autolog import __MLflowTfKeras2Callback, _TensorBoard
 
-    tb = _get_tensorboard_callback(lst)
+    tb = _get_tensorboard_callback(callbacks)
+    for callback in callbacks:
+        if isinstance(callback, MLflowCallback):
+            raise ValueError(
```

Comment:
Can we change this to an `MLflowException` that specifies a 4xx type error for a configuration issue?

Reply:
good call, done!